### PR TITLE
document for api get comments in book

### PR DESCRIPTION
**Pull request by mvn-caonguyen-dn**
## Description
document for api get comments in book
## Reference Requirements
- Product Backlog ID: PBI-39
- Task ID: #39.4

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- resources/views/api-doc-builders/index.blade.php

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
LGTM

File: docs/api-references/comment.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| next | String | Link of next page |
+
+
+```json
+{
+    "meta": {
+        "status": "successfully",
+        "code": 200
+    },
+    "data": [
```
Comment (at-huydq) [inline docs/api-references/comment.md]: a ko thấy parent id nhỉ ? phải có để biết phân cấp chứ

File: docs/api-references/comment.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            "team": "SA",
+            "avatar_url": "http://flappybook.tech/tmp/7c5266db2cd916e1af404a688980e4dd.jpg",
+            "created_at": "2018-02-08 16:34:10",
+            "updated_at": "2018-02-08 16:34:10",
+            "deleted_at": null
+        },
+        {
+            "id": 15,
+            "comment": "Vel natus quo explicabo cupiditate autem dolor et aliquid.",
+            "username": "Mr. Morris Glover V",
```
Comment (at-huydq) [inline docs/api-references/comment.md]: ở trên là `name` mà sao dưới này là `username` ?

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**